Add JMS bbes (#4880)
ballerina-platform/ballerina-distribution: examples/jms-consumer-acknowledgement/jms_consumer_acknowledgement.md

# JMS message producer - Manual acknowledgment

> The `jms:MessageConsumer` allows manual acknowledgment for received messages. A `jms:MessageConsumer` can be initialized by using a `jms:Session` object. To configure manual acknowledgment mode,  create a `jms:Session` object with acknowledge mode set to `jms:CLIENT_ACKNOWLEDGE`.

Reviewer: ```suggestion
The `jms:MessageConsumer` allows manual acknowledgment for received messages. A `jms:MessageConsumer` can be initialized by using a `jms:Session` object. To configure manual acknowledgment mode, create a `jms:Session` object with the acknowledge mode set to `jms:CLIENT_ACKNOWLEDGE`.
```

::: code jms_consumer_acknowledgement.bal :::

## Prerequisites

> - Start a [ActiveMQ broker](https://activemq.apache.org/getting-started) instance.

Reviewer: ```suggestion
- Start a [ActiveMQ broker](https://activemq.apache.org/getting-started) instance.
```
We don't need a bullet point if we have only one point.

Author: Fixed with [29db9a5527a2a51e3c7bd84cb993a17d1558c238](https://github.com/ballerina-platform/ballerina-distribution/pull/4880/commits/29db9a5527a2a51e3c7bd84cb993a17d1558c238)

> Run the program by executing the following command.

Reviewer: ```suggestion
Run the program by executing the following command.
```
Should this also be under the `Prerequisites` section?

Author: Nope

::: out jms_consumer_acknowledgement.out :::

> >**Tip:** Run the JMS message producer given in the [JMS message producer - Produce message](/learn/by-example/jms-producer-produce-message-send) example to produce some messages to the queue.

Reviewer: ```suggestion
>**Tip:** Run the JMS message producer given in the [JMS message producer - Produce message](/learn/by-example/jms-producer-produce-message-send) example to produce a few sample messages to the queue.
```
